pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -445,6 +446,14 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
     )
     private String clusterName;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "Defines how the broker will anonymize the role and originalAuthRole before logging. "
+                    + "Possible values are: NONE (no anonymization), REDACTED (replaces with '[REDACTED]'), "
+                    + "hash:SHA256 (hashes using SHA-256), and hash:MD5 (hashes using MD5). Default is NONE."
+    )
+    private String authenticationRoleLoggingAnonymizer = "NONE";
```

Comment:
please add this to `conf/broker.conf` and `conf/standalone.conf` so that `PULSAR_PREFIX_` wouldn't be needed in k8s deployments.